integration_tests/setup_db.py

```diff
@@ -303,3 +339,26 @@ def _setup_document_data(
         )
         test_db.add(fd)
         test_db.flush()
+
+
+def _setup_event_data(
+    test_db: Session,
+    family_id: str,
+    configure_empty: bool = False,
+) -> None:
+    if configure_empty is True:
+        return None
+
+    for index in range(EXPECTED_NUM_EVENTS):
+        data = EXPECTED_EVENTS[index]
+        fe = FamilyEvent(
+            import_id=data["import_id"],
+            title=data["event_title"],
+            date=data["date"],
+            event_type_name=data["event_type_value"],
+            family_import_id=data["family_import_id"],
+            # family_document_import_id=data["family_document_import_id"],
```

Comment:
So we don't test events associated with a FamilyDocument? Might be worth a TODO or a NOTE here?

Reply:
Noted!